flow/designs/gf12/ariane/metadata-base-ok.json

```diff
@@ -3,217 +3,195 @@
+  "cts__power__switching__total__pre_repair": 0.0538759,
+  "cts__power__total": 0.102252,
+  "cts__power__total__post_repair": 0.102202,
+  "cts__power__total__pre_repair": 0.102143,
+  "cts__route__wirelength__estimated": 2823450.0,
+  "cts__runtime__total": "2:35:12",
   "cts__timing__drv__hold_violation_count": 0,
-  "cts__timing__drv__hold_violation_count__post_repair": 8682,
-  "cts__timing__drv__hold_violation_count__pre_repair": 8631,
+  "cts__timing__drv__hold_violation_count__post_repair": 1373,
+  "cts__timing__drv__hold_violation_count__pre_repair": 1413,
   "cts__timing__drv__max_cap": 0,
   "cts__timing__drv__max_cap__post_repair": 0,
   "cts__timing__drv__max_cap__pre_repair": 0,
-  "cts__timing__drv__max_cap_limit": 0.0995229,
-  "cts__timing__drv__max_cap_limit__post_repair": 0.150428,
-  "cts__timing__drv__max_cap_limit__pre_repair": 0.150428,
+  "cts__timing__drv__max_cap_limit": 0.0330308,
+  "cts__timing__drv__max_cap_limit__post_repair": 0.0351715,
+  "cts__timing__drv__max_cap_limit__pre_repair": 0.0351715,
   "cts__timing__drv__max_fanout": 0,
   "cts__timing__drv__max_fanout__post_repair": 0,
   "cts__timing__drv__max_fanout__pre_repair": 0,
   "cts__timing__drv__max_fanout_limit": 0,
   "cts__timing__drv__max_fanout_limit__post_repair": 0,
   "cts__timing__drv__max_fanout_limit__pre_repair": 0,
-  "cts__timing__drv__max_slew": 76,
-  "cts__timing__drv__max_slew__post_repair": 32,
-  "cts__timing__drv__max_slew__pre_repair": 32,
-  "cts__timing__drv__max_slew_limit": -0.200575,
-  "cts__timing__drv__max_slew_limit__post_repair": -0.200577,
-  "cts__timing__drv__max_slew_limit__pre_repair": -0.200577,
-  "cts__timing__drv__setup_violation_count": 0,
-  "cts__timing__drv__setup_violation_count__post_repair": 2091,
-  "cts__timing__drv__setup_violation_count__pre_repair": 2092,
-  "cts__timing__setup__tns": 0,
-  "cts__timing__setup__tns__post_repair": -313060,
-  "cts__timing__setup__tns__pre_repair": -313140,
-  "cts__timing__setup__ws": 6.11844,
-  "cts__timing__setup__ws__post_repair": -328.294,
-  "cts__timing__setup__ws__pre_repair": -328.374,
-  "cts_fill__cpu__total": 9.04,
-  "cts_fill__mem__peak": 1389568.0,
-  "cts_fill__runtime__total": "0:10.01",
-  "design__io__hpwl": 438280700,
-  "detailedplace__cpu__total": 140.09,
+  "cts__timing__drv__max_slew": 22,
+  "cts__timing__drv__max_slew__post_repair": 29,
+  "cts__timing__drv__max_slew__pre_repair": 30,
+  "cts__timing__drv__max_slew_limit": -0.418555,
+  "cts__timing__drv__max_slew_limit__post_repair": -0.254846,
+  "cts__timing__drv__max_slew_limit__pre_repair": -1.01813,
+  "cts__timing__drv__setup_violation_count": 1561,
+  "cts__timing__drv__setup_violation_count__post_repair": 3222,
+  "cts__timing__drv__setup_violation_count__pre_repair": 3180,
+  "cts__timing__setup__tns": -114433,
+  "cts__timing__setup__tns__post_repair": -778658,
+  "cts__timing__setup__tns__pre_repair": -756638,
+  "cts__timing__setup__ws": -177.325,
```

Comment:
all of these are significantly worse

Reply:
These changes are mainly caused by the update in the sdc file. Previously, we had unrealistic values there, and with the new sdc we got worse results.